app/search_builders/hyrax/exposed_models_relation.rb

```diff
@@ -0,0 +1,8 @@
+module Hyrax
+  # A relation that scopes to all user visible models (e.g. works + collections + file sets)
+  class ExposedModelsRelation < AbstractTypeRelation
+    def allowable_types
+      Hyrax.config.curation_concerns + [Collection, ::FileSet]
```

Comment:
We've never exposed AdminSets via resource synch before, but we have Collections. I'm trying to be consistent with what is already there.